egs/gigaspeech/ASR/local/compute_fbank_gigaspeech_splits.py

```diff
@@ -100,6 +101,8 @@ def compute_fbank_gigaspeech_splits(args):
     extractor = KaldifeatFbank(KaldifeatFbankConfig(device=device))
     logging.info(f"device: {device}")
 
+    set_audio_duration_mismatch_tolerance(0.01)  # 10ms tolerance
```

Comment:
You may need to set it to 0.1

Reply:
Did you see any error if set 0.01? I just finished extraction by setting 0.01

Reply:
Yes, I remember that I have to reset it to 0.1. But it is fine if 0.01 works for you.